app/graphql/resolvers/invoice_credit_notes_resolver.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+module Resolvers
+  class InvoiceCreditNotesResolver < Resolvers::BaseResolver
+    include AuthenticableApiUser
+    include RequiredOrganization
+
+    description "Query invoice's credit note"
+
+    argument :invoice_id, ID, required: true, description: 'Uniq ID of the invoice'
+    argument :page, Integer, required: false
+    argument :limit, Integer, required: false
+
+    type Types::CreditNotes::Object.collection_type, null: true
+
+    def resolve(invoice_id: nil, page: nil, limit: nil)
+      validate_organization!
+
+      Invoice.find(invoice_id)
+        .credit_notes
```

Comment:
Are we sure we want to return all credit notes here?

Even the ones not available or already voided?

Reply:
Good point, checking

Reply:
Apparently no ! We need to display all credit notes at this point